de/_posts/2019-09-21-code-and-cake.markdown

```diff
@@ -0,0 +1,21 @@
+---
+tags: intermediate code-cake
+categories: de events
+i18n_key: code_cake_2019_09
+---
+
+## Lerne noch mehr Programmieren ♥️
+
+Egal ob du an einem Beginner Workshop teilgenommen hast oder dir selber programmieren beibringst, Code & Cake Workshop ist der perfekte Tag für dich um weiter zu lernen.
```

Comment:
Here we write "Code & Cake Workshop" without an article (which I feel like does not sound so German) and in line 21 we write "Code&Cake" without spaces, article or "Workshop". I would prefer some kind of consistency and my suggestion is either say "Code & Cake" as a name of the event (without article, with spacing, without "Workshop") or call it "das Code & Cake Event" but stick to this version. I also would not see the "Code & Cake" as a workshop and would rather call it "Event" or "Projekttag" or something like that.